.gitignore

```diff
@@ -46,3 +46,4 @@
 # .bak files
 *.bak
 /vendor/cache/**/
+/db/development.sqlite
```

Comment:
we already ignore all .sqlite files in that folder

Reply:
ah, it didn't match the name in the docker-compose.yml, i've updated it.